src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -9,16 +9,19 @@ import { shell, remote } from 'electron';
 import * as actions from '../../actions';
 import { GUPPY_REPO_URL } from '../../constants';
 import { isMac } from '../../services/platform.service';
+import {
+  openProjectInFolder,
+  openProjectInEditor,
+} from '../../services/shell.service';
 import { getSelectedProject } from '../../reducers/projects.reducer';
 import { getDevServerTaskForProjectId } from '../../reducers/tasks.reducer';
 
-import type { Task } from '../../types';
+import type { Project, Task } from '../../types';
 
 const { app, process, Menu } = remote;
 
 type Props = {
-  selectedProject: ?string,
-  selectedProjectId: ?string,
```

Comment:
Bit of cleanup. I realized we don't need the `id` if we have the entire project.